apps/wpcom-block-editor/src/wpcom/features/tracking/site-editor-load.tsx

```diff
@@ -0,0 +1,41 @@
+import { select, useSelect } from '@wordpress/data';
+import { registerPlugin } from '@wordpress/plugins';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import { useEffect } from 'react';
+import tracksRecordEvent from './track-record-event';
+
+const { unlock } = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
```

Comment:
Let's use try-catch to avoid unlock api is unavailable 🙂

See https://github.com/Automattic/wp-calypso/blob/b0f4e1b4e4547db5aa8f9fa8062a92056ed3f825/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/block-editor-nux.js#L26

Reply:
In https://github.com/Automattic/wp-calypso/pull/79663, you mentioned, "_the new version allows it by default but it's not the version we use in this scenario_".

Considering the `@wordpress/plugins` module seems to be now up-to-date (https://github.com/WordPress/gutenberg/blob/6830aecbd81ede7d1502e67df9eb6b5b0689e0c0/packages/plugins/package.json#L3, [package.json#L96](https://github.com/Automattic/wp-calypso/blob/9d38fcbb337bfadb632ed0e50d0ca1971cce6846/apps/editing-toolkit/package.json#L96)), I believe we might not need to catch the error anymore (CMIIW). What do you think? 👀

Reply:
Seems this is the plugin of the Gutenberg, so the version might depend on that site. But I'm not very sure.

Reply:
Let's add the try/catch, better to be safe than sorry 🙂

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/81335/commits/cd1c44abe719638649a468e30d2f2bd1600664b8 with a comment ✔️